src/components/Pagination/Link.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { createElement as E, Component } from 'react'
+import T from 'prop-types'
+import { classNames } from '../../utils/'
+
+class PaginationLink extends Component {
+  handleClick = (e) => {
+    this.props.callback(e, this)
+  }
+
+  render() {
+    const {
+      to,
+      component,
+      node,
+      className,
+      isActive,
+      activeClassName = 'is-active',
+      callback,
+      children,
+      ...rest
+    } = this.props
+
+    const onClickProps = callback
+      ? {
+        onClick: this.handleClick
+      } : {}
+
+    const ComponentOverride = component
+
+    if (ComponentOverride) {
+      return (
```

Comment:
Good.  This will allow consumers to use React router `<Link />` component with these.